docs/installation/manage/_index.md

```diff
@@ -147,6 +147,47 @@ brew services restart viam-server
 {{% /tab %}}
 {{< /tabs >}}
 
+## Update `viam-server`

```

Comment:
Make sure to add an alias to the to the old update page then I think this is good to go

Reply:
Oops, thank you!! On it.